plugins/cli/Cargo.toml

```diff
@@ -13,6 +13,14 @@ links = "tauri-plugin-cli"
 rustc-args = ["--cfg", "docsrs"]
 rustdoc-args = ["--cfg", "docsrs"]
 
+[package.metadata.platforms.support]
+windows = { level = "full", notes = "" }
```

Comment:
Windows does not support cli output without workarounds

Reply:
Here's the insane writeup i mentioned https://github.com/tauri-apps/tauri/issues/8305#issuecomment-1826871949

Reply:
We maybe should mention this insane thing in the docs of the plugin. Is the guide a good place for it to link to it or copy it?

Reply:
hmm, yeah. i'll add it to the guide then.